src/components/Content/TaskHierarchy/Task/Task.js

```diff
@@ -35,7 +35,10 @@ class Task extends Component {
   }
 
   redirectToDiscuss = () => {
-    // this.props.history.push('/pursuance/1/discuss');
+    const { history, taskData, match: { params: { pursuanceId } } } = this.props;
+    history.push({
+      pathname: `/pursuance/${pursuanceId}/discuss/task/${taskData.gid}`
+    });
```

Comment:
@Moe-Shoman Hmm maybe I'm not running the latest code, because it looks like you already made it so clicking this icon _does_ take the user to the chat room...

Reply:
@elimisteve do you have a LeapChat server running?

Reply:
I was viewing Pursuance through a fucked up tunnel; my bad. I see the room now :+1: . Sick.

Reply:
Cool!